docs/_sass/_reboot.scss

```diff
@@ -0,0 +1,8 @@
+/*# sourceMappingURL=bootstrap-reboot.min.css.map */
```

Comment:
Are we actually generating a source map somewhere?

Reply:
Not that I'm aware of.

Does cURL permit something like `sourcemap=none` which would suppress this? I've used it when converting by hand, but I'm not sure it applies to a process like Rake tasks 🤔

Reply:
Oh, I see: This is from the pre-generated CSS file that we are downloading.
No, not much we can do about this, I think.

Reply:
@pathawks Yeah, given how this is structured, it's redundant.